hpx won't build on POWER9 with clang 8

## Expected Behavior

Normal compilation

## Actual Behavior

compile error (could be just a clang 8.0.1 problem, not POWER9 related):

```bash
/home/users/khuck/src/phylanx/tools/buildbot/src/hpx/libs/serialization/include/hpx/serialization/traits/brace_initializable_traits.hpp:48:59: error: 
      no member named 'index_sequence' in namespace 'std'
        inline constexpr auto is_brace_constructible(std::index_sequence<I...>,
                                                     ~~~~~^
```

## Steps to Reproduce the Problem

... Please be as specific as possible while describing how to reproduce your problem.

  1.  cmake -DCMAKE_CXX_COMPILER=/packages/llvm/8.0.1/bin/clang++ -DCMAKE_C_COMPILER=/packages/llvm/8.0.1/bin/clang -DCMAKE_BUILD_TYPE=Release -DBOOST_ROOT=/home/users/khuck/src/phylanx/tools/buildbot/build-cyclops-ppc64le-Linux-clang/boost-1.65.1 -DHPX_WITH_MALLOC=system -DHWLOC_ROOT= -DCMAKE_INSTALL_PREFIX=. -DHPX_WITH_THREAD_IDLE_RATES=ON -DHPX_WITH_PARCELPORT_MPI=ON -DHPX_WITH_PARCEL_COALESCING=OFF -DHPX_WITH_TOOLS=OFF -DHPX_WITH_TESTS=ON -DHPX_WITH_EXAMPLES=OFF -DHPX_WITH_THREAD_MANAGER_IDLE_BACKOFF=OFF -DHPX_WITH_APEX=TRUE -DAPEX_WITH_ACTIVEHARMONY=TRUE -DACTIVEHARMONY_ROOT=/usr/local/packages/activeharmony/4.6.0-ppc64le -DAPEX_WITH_OTF2=TRUE -DOTF2_ROOT=/usr/local/packages/otf2/2.0-ppc64le -DAPEX_WITH_PAPI=TRUE -DPAPI_ROOT=/usr/local/packages/papi/5.5.1 -DHPX_WITH_APEX_NO_UPDATE=FALSE -DHPX_WITH_APEX_TAG=develop -DHWLOC_ROOT=/usr/local/packages/hwloc/1.8 -DHPX_WITH_RDTSCP=OFF -DHPX_WITH_DEPRECATION_WARNINGS=Off -DHPX_WITH_MAX_CPU_COUNT=160 /home/users/khuck/src/phylanx/tools/buildbot/src/hpx
  1.  make -j40

## Specifications

... Please describe your environment

  - HPX Version:  current master
  - Platform (compiler, OS):  POWER9, Clang 8.0.1

--- a/libs/serialization/include/hpx/serialization/traits/brace_initializable_traits.hpp
+++ b/libs/serialization/include/hpx/serialization/traits/brace_initializable_traits.hpp
@@ -15,6 +15,7 @@
 
 #include <cstddef>
 #include <type_traits>
+#include <utility>
 
 namespace hpx { namespace traits {
 

